### PR TITLE
D8 un 688

### PR DESCRIPTION
**Pull request by DuttonMa**
Various post release fixes for Urology
- Config split was not working properly for Fastly or Google Analytics - has now been redone
- Favicons included unnecessary html tags that were causing problems in the headers

### CODE REVIEW

File: config/urologyservicesinquiry/config/block.block.exposedformnews_searchnews_search_page.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 uuid: 64397037-46b6-4036-af65-86413bbc47d8
 langcode: en
-status: true
+status: false
```
Comment (ooneill547) [inline config/urologyservicesinquiry/config/block.block.exposedformnews_searchnews_search_page.yml:3]: Were the news search and facets meant to be removed?

File: config/urologyservicesinquiry/config/core.extension.yml [hunk 2]
```diff
@@ -108,7 +109,6 @@ module:
   unity_internal_link_checker: 0
   unity_map_embed: 0
   unity_search_pages: 0
-  urologyservicesinquiry_breadcrumbs: 0
```
Comment (ooneill547) [inline config/urologyservicesinquiry/config/core.extension.yml:111]: I'm not sure this module should have been removed, it would break the breadcrumbs

**Review by ooneill547 — COMMENTED** [review]
Just a few queries

File: config/urologyservicesinquiry/config/block.block.exposedformnews_searchnews_search_page.yml (hunk 1 above)
Comment (DuttonMa) [reply config/urologyservicesinquiry/config/block.block.exposedformnews_searchnews_search_page.yml]: Good question, have added them back in

File: config/urologyservicesinquiry/config/core.extension.yml (hunk 2 above)
Comment (DuttonMa) [reply config/urologyservicesinquiry/config/core.extension.yml]: Have installed it now

**Review by ooneill547 — APPROVED** [review]
Looks good

**Pull request merged**